sw/banshee/src/engine.rs

```diff
@@ -571,9 +587,12 @@ pub unsafe fn add_llvm_symbols() {
 
 impl CpuState {
     /// Create a new CpuState
-    pub fn new(num_dm: usize) -> Self {
+    pub fn new(num_dm: usize, hartid: usize, bootrom_addr: u32) -> Self {
+        let mut reg_init: [u32; 32] = [0; 32];
+        reg_init[10] = hartid as u32;
+        reg_init[11] = bootrom_addr;
```

Comment:
This should come from the config eventually. I am fine with merging this like this for now, but let's create an issue to make sure we make this parameterizable in the future.

Reply:
I created an Issue here for now: #350